.github/pull_request_template.md

```
@@ -1,34 +1,339 @@
+- [ ] Monitoring dashboards updated
+
+---
+
+## 💬 Further Comments
+
+_[If this is a relatively large or complex change, explain your approach, alternatives you considered, design decisions, etc.]_
+
+### Design Decisions
+
+_[Explain key design decisions and tradeoffs]_
+### Alternatives Considered
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)